src/cpp/src/tokenizer.cpp

```diff
@@ -345,9 +381,19 @@ class Tokenizer::TokenizerImpl {
             {"bos_token",  m_bos_token},
             {"eos_token", m_eos_token},
             {"pad_token", m_pad_token},
+            {"system_message", system_message.empty() ? jinja2::EmptyValue() : jinja2::Value{system_message}},
             {"add_generation_prompt", add_generation_prompt},
         };
-        return tpl.RenderAsString(params).value();
+        std::string history_txt;
+        try {
+            history_txt = tpl.RenderAsString(params).value();
+        } catch (const std::exception& error) {
+            OPENVINO_THROW("chat_template is not supported please "
```

Comment:
can we write that it's not supported for this specific model? to avoid confusion that it's not supported at all

E.g. we can also write that it's not supported as JinjaCpp limitation.

Reply:
done